### ISSUE TITLE
Error during bower installation

### ISSUE THREAD

**Issue by wymmeroo** [alexcrack/angular-ui-notification#35]
Hi,
I tried to install using bower with the following command:
bower install angular-ui-notification --save

It looks like that your bower.json file in not valid anymore, i get the following error message:

Additional error details:
The "main" field cannot contain minified files

Regards
Tobias

**Comment by alexcrack** [comment]
I just installed module using bower without any errors.

```
bower install angular-ui-notification --save
bower angular-ui-notification#* cached git://github.com/alexcrack/angular-ui-notification.git#0.0.11
bower angular-ui-notification#*         validate 0.0.11 against git://github.com/alexcrack/angular-ui-notification.git#*
bower angular-ui-notification#~0.0.11     cached git://github.com/alexcrack/angular-ui-notification.git#0.0.11
bower angular-ui-notification#~0.0.11   validate 0.0.11 against git://github.com/alexcrack/angular-ui-notification.git#~0.0.11
bower angular-ui-notification#~0.0.11    install angular-ui-notification#0.0.11

angular-ui-notification#0.0.11 bower_components/angular-ui-notification
```

Which version of bower do you use?
Give me information about your system, please: node version, OS.

**Comment by wymmeroo** [comment]
I use bower 1.3.2 and windows 8.1

In the bower manual, under the main section, they state that no minified files should be referenced in the main section. Could this be the problem? Which version do you use?

From:
http://bower.io/docs/creating-packages/

Image and font files may be used or referenced within the JS or Sass files, but are not main files as they are not entry-points.

    Use source files with module exports and imports over pre-built distribution files.
    Do not include minified files.
    Do not include assets files like images, fonts, audio, or video.
    Filenames should not be versioned (Bad: package.1.1.0.js; Good: package.js).
    Globs like js/*.js are not allowed.

**Comment by alexcrack** [comment]
Fixed in v0.1.0
